### PR TITLE
Update AWX images to CentOS 8

### PR DESCRIPTION
**Pull request by shanemcd**

### CODE REVIEW

File: installer/roles/image_build/templates/Dockerfile.j2 [hunk 1]
```diff
@@ -1,75 +1,56 @@
-FROM centos:7
+FROM centos:8
 
 USER root
 
-ADD ansible.repo /etc/yum.repos.d/ansible.repo
```
Comment (shanemcd) [inline installer/roles/image_build/templates/Dockerfile.j2:5]: Had to switch to pip because we don't have upstream EL8 RPMs for core yet.

Comment (wenottingham) [reply installer/roles/image_build/templates/Dockerfile.j2:5]: EPEL?

File: installer/roles/image_build/templates/Dockerfile.j2 [hunk 2]
```diff
@@ -80,30 +61,19 @@ ADD requirements/requirements_ansible.txt \
...
-  nodejs \
-  xmlsec1-devel \
-  xmlsec1-openssl-devel
+  nodejs
 
 # makes postgres available for data migrations when upgrading
 RUN export PATH=/usr/pgsql-10/bin:$PATH
 
-RUN yum -y clean all
+RUN dnf -y clean all
```
Comment (wenottingham) [inline installer/roles/image_build/templates/Dockerfile.j2:75]: Applies to the dev env as well, but this perhaps should be moved to after installing kubectl/oc.

File: installer/roles/image_build/templates/Dockerfile.j2 (hunk 1 above)
Comment (shanemcd) [reply installer/roles/image_build/templates/Dockerfile.j2:5]: @wenottingham what does that actually get us vs just installing from pip?

File: installer/roles/image_build/templates/Dockerfile.j2 (hunk 2 above)
Comment (shanemcd) [reply installer/roles/image_build/templates/Dockerfile.j2:75]: Will make a separate PR later and do this in both places.

File: installer/roles/image_build/templates/Dockerfile.j2 (hunk 1 above)
Comment (wenottingham) [reply installer/roles/image_build/templates/Dockerfile.j2:5]: Closer to anything a downstream might do, at least.

Comment (wenottingham) [reply installer/roles/image_build/templates/Dockerfile.j2:5]: It's what the dev env does at the moment

Comment (shanemcd) [reply installer/roles/image_build/templates/Dockerfile.j2:5]: @wenottingham Seems like you care more than I do, made the change.

**Comment by shanemcd** [comment]
image

**Review by ryanpetrello — APPROVED** [review]

**Pull request merged**